### DISCUSSION TITLE
Feedback on new features

### DISCUSSION THREAD

**Discussion by lynnpais** [broadinstitute/seqr#2688] [Ideas]
Hi Hana, I would like to get your input on how much effort it would be to implement two new features:

1. Improve notifications to an analyst about which cases are due for re-analysis
 A new metric 'Due for re-analysis' which filters cases by the last date the 'Analyzed By' was selected. Selecting 'Due for re-analysis' would return all cases that have completed one year since the Analyzed By WES/WGS field was last selected by the analyst.

2. Autogenerate/suggest text in the variant notes field
It would be incredible helpful if I could save sentences such "Phenotype not relevant" or "Allele frequency not consistent with prevalence" etc., without having them to type them out each time. I use these phrases anywhere between 5-10 times per case. Would it be possible for an analyst to save text somewhere and use it in the notes field by typing a keyword or something of the sort? For example, in clinic, when I type '.Intro' it generates the general introduction template which I then edit. 

Thanks for considering.

**Comment by hanars** [comment]
1) That would be pretty simple. I assume you would want it added as a filter for the list of families on the project page, the way we have the "analysed by me" or "data loaded" filters now

2) This one is a bit harder. It is probably possible, but it is potentially a lot of work. The difficulty also depends on how customizable it needs to be - are you envisioning it that everyone could configure their own autogeneration preferences? Or would seqr have them pre-defined, but then everyone would have to agree on which ones they want? I'm also not really sure that this is the best solution to the problem you are seeming to have. Autocomplete makes a lot of sense for things like an introduction template, where you have to always fill in the same types of information but the info itself will be different every time. If you are always making the same notes with the same content, that strikes me as more of a tag than a note. The examples you gave are not the sort of thing that belongs in our current tag list, but maybe they would belong in some new field like "analysis tags" where you could pick from a list of common things you find during your analysis
